src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -82,6 +82,8 @@ def __init__(self, config):
 
         self.useDsetReplicaDeep = getattr(config.RucioInjector, "useDsetReplicaDeep", False)
         self.delBlockSlicesize = getattr(config.RucioInjector, "delBlockSlicesize", 100)
+        self.blockDeletionDelayHours = getattr(config.RucioInjector, "blockDeletionDelayHours", 0)
```

Comment:
Just a note, you don't need to update the code, but given that you don't use this variable (blockDeletionDelayHours) anywhere in the class, you could simply remove the self (not making it an object instance attribute)